src/eventshub/src/component/SearchFilter/CustomRefinementList.tsx

```diff
@@ -50,9 +51,8 @@ const CustomRefinementList = ({ title, options }:CustomRefinementListProps) => {
           <Heading as='h3' size="base" className="er-m-0">
             {title}
           </Heading>
-          <ChevronDownIcon
-            className={cnb('er-w-1em', isOpen && 'er-rotate-180')}
```

Comment:
I stopped troubleshooting why the rotate wasn't working. It was being applied correctly but the UI didn't update.